src/Microsoft.Azure.WebJobs.Host/Bindings/ConverterManager.cs

```diff
@@ -226,6 +226,11 @@ private static OpenType GetOpenType(Type t)
                 }
             }
 
+            if (typeDest == typeof(JArray) && typeof(IEnumerable).IsAssignableFrom(typeSource))
```

Comment:
I think in that case, it involves chaining & composing converters. If the user defines a T-->S converter, and we can bind to IEnumerable<T>, then we could also bind to IEnumerable<S>. , but that may be invoking the converter. 
But this can get tricky quick - so I'm a little shy to get into that unless we have a clear case to guide us. 




---
In reply to: [102849948](https://github.com/Azure/azure-webjobs-sdk/pull/1037#discussion_r102849948) [](ancestors = 102849948)